test/schemas/store-order-recipient.test.js

```diff
@@ -2,13 +2,15 @@ const faker = require('faker');
 const config = require('../../lib/config');
 
 const { storeOrderRecipientSchema, isNumberOrUuid } = require('../../lib/schemas/store-order-recipient');
+const { mockConfig } = require('../test-helpers/mock-config');
 
 const testLocale = faker.random.locale();
 const testNumber = faker.random.number();
 
 describe('storeOrderRecipientSchema.validate()', () => {
   beforeEach(() => {
-    config.set('supportedLocales', [testLocale]);
+    config.reset();
```

Comment:
I see this is a shorthand, but for safety, `config.reset()` should go into a `afterEach`